src/services/credentialEvents/index.ts

```diff
@@ -58,6 +60,28 @@ export default class CredentialEvents {
       return
     }
 
+    if (this.isCredentialError(record)) {
+      this.logger.debug('There was an erorr in credential issuance of credential %s', record.id)
+      if (!record.errorMessage) {
+        this.logger.debug('Errror message in error report is missing for credential', record.id)
+        return
+      }
+
+      const startIndex = record.errorMessage.indexOf('{')
```

Comment:
this is extremely evil. Is the cloudagent prepending something?

Reply:
yes it looks like this: ` errorMessage: 'issuance-abandoned: {"message":"Invalid pin detected in credential proposal for connection db3daef6-f6f1-4c7f-b3e8-512f02fb447c","pinTries":4}'`